script/check-config.sh

```diff
@@ -212,11 +213,11 @@ flags=(
 )
 check_flags "${flags[@]}"
 
-if [ "$kernelMajor" -lt 5 ] || [ "$kernelMajor" -eq 5 -a "$kernelMinor" -le 1 ]; then
+if [ "$kernelMajor" -lt 5 ] || [[ "$kernelMajor" -eq 5 && "$kernelMinor" -le 1 ]]; then
```

Comment:
I think you're probably going to like my `[[` avoidance even less here than usual :innocent: :sweat_smile:

```suggestion
if [ "$kernelMajor" -lt 5 ] || { [ "$kernelMajor" -eq 5 ] && [ "$kernelMinor" -le 1 ]; }; then
```